src/unix/bsd/apple/mod.rs

```diff
@@ -19,6 +19,35 @@ pub type fsfilcnt_t = ::c_uint;
 pub type speed_t = ::c_ulong;
 pub type tcflag_t = ::c_ulong;
 pub type nl_item = ::c_int;
+pub type natural_t = ::c_uint;
+pub type integer_t = ::c_int;
+pub type mach_port_t = ::natural_t;
+pub type thread_t = ::mach_port_t;
+pub type mach_msg_type_number_t = ::natural_t;
+pub type thread_policy_flavor_t = ::natural_t;
+pub type kern_return_t = ::c_int;
+pub type thread_latency_qos_t = ::integer_t;
+pub type thread_throughput_qos_t = ::integer_t;
+// libc-test does not like structs
+//pub type thread_policy_t = *mut integer_t;
+//pub type thread_standard_policy_data_t = thread_standard_policy;
+//pub type thread_extended_policy_data_t = thread_extended_policy;
+//pub type thread_time_constraint_policy_data_t = thread_time_constraint_policy;
+//pub type thread_precedence_policy_data_t = thread_precedence_policy;
+//pub type thread_affinity_policy_data_t = thread_affinity_policy;
+//pub type thread_background_policy_data_t = thread_background_policy;
+//pub type thread_latency_qos_policy_data_t = thread_latency_qos_policy;
+//pub type thread_throughput_qos_policy_data_t = thread_throughput_qos_policy;
+// libc-test does not like mutable pointers
+//pub type thread_standard_policy_t = * mut thread_standard_policy_data_t;
+//pub type thread_extended_policy_t = * mut thread_extended_policy_data_t;
```

Comment:
What's up with these comments? Could you gist the errors you were seeing from libc-test?